src/main/java/com/ironpanthers/lib/Util.java

```diff
@@ -0,0 +1,44 @@
+package com.ironpanthers.lib;
+
+public class Util {
```

Comment:
Does this exist twice now? Once here, and once in the non com? Possible artifact of switching back to not using a different library.

Reply:
Oops yeah let's drop this one. It becomes an issue if you import both Utils, yeah